### PR TITLE
Fix action calculation

### PR DESCRIPTION
**Pull request by MarcelBruckner**

### CODE REVIEW

**Comment by marcelbrucker** [comment]
Looks good to me.

Only too things:
1. Why this change?
Bildschirmfoto 2020-08-02 um 12 30 11
2. Some length assertion in load_expert_trajectories_tests.py fails in CI. Is this valid or a problem?

File: bark_ml/library_wrappers/lib_tf_agents/runners/sac_runner.py [hunk 1]
```diff
@@ -60,24 +81,35 @@ def GenerateExpertTrajectories(self, num_trajectories: int = 1000, render: bool
...
-        expert_trajectories['obs'].append(state)
+        state_not_norm = self.GetStateNotNormalized()
+        expert_trajectories['obs_norm'].append(state)
+        expert_trajectories['obs'].append(state_not_norm)
         expert_trajectories['act'].append(action_step.action.numpy())
 
         if render:
           self._environment.render()
 
       if info and info['goal_reached']:
```
Comment (MarcelBruckner) [inline bark_ml/library_wrappers/lib_tf_agents/runners/sac_runner.py:113]: I added the info check to filter the expert trajectories that end with goal reached. Only if the environment gives a valid last step (The workaround for the index error) and the goal was reached, we add the trajectory to the set of expert trajectories.
The change with `# if True` was just for debugging, so I didn't have to wait for a converged agent :)

**Comment by MarcelBruckner** [comment]
Also the CI is fixed in the meantime :+1:

**Comment by MarcelBruckner** [comment]
I think we can merge this now. What is your opinion @marcelbrucker ?
If you approve it I will do the merge later today.

File: examples/example_params/gail_params.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "ML": {
         "Settings": {
             "GPUUse": 0,
-            "NormalizeFeatures": true
+            "NormalizeFeatures": true,
+            "SAC":true
```
Comment (marcelbrucker) [inline examples/example_params/gail_params.json]: I believe this SAC parameter is a remnant of @marvinlue and can be deleted.

**Review by marcelbrucker — COMMENTED** [review]
Apart from this unnecessary SAC parameter in gail_params.json I approve this pull request.

File: examples/example_params/gail_params.json (hunk 2 above)
Comment (MarcelBruckner) [reply examples/example_params/gail_params.json]: Removed it :+1:

**Pull request merged**